adatest/_topic_model.py

```diff
@@ -157,8 +157,9 @@ def __init__(self, topic, test_tree):
         else:
             
             # we are in a highly overparametrized situation, so we use a linear SVC to get "max-margin" based generalization
-            self.model = CVModel()
-            self.model.fit(embeddings, labels)
+            self.model = ConstantModel(1.0)

```

Comment:
Why is this switched? it seems like it will disable all topic learning :)

Reply:
Glad you found this, I was getting errors there for some reason so I made that change to temporarily unblock my work. I shouldn't have checked it in though.

Reply:
@slundberg Reverted and tested again with no issues. Thanks!